packages/ui-params/src/valueToText.tsx

```diff
@@ -150,6 +151,12 @@ function valueToText (type: string, value: any, swallowError: boolean = true, co
   //   }
   // }
 
+  // When displaying parameters of type Hash, these will sometimes come in the
```

Comment:
Ahh, ok. So basically what happens - when it comes from external sources (i.e. via the API), it would be a Codec type. (So Hash). However the UI actually captures these as Uint8Array, hence the discrepancy.